src/components/DeckMechanisms/index.js

```diff
@@ -422,8 +444,8 @@ export default class DeckMechanisms extends React.Component {
     }
 
     if (this.state.turn === 1) {
-      const unplayableSpells = ['W1', 'S10', 'N15']
-      
+      const unplayableSpells = ['W1', 'S10', 'N15', 'N63']
```

Comment:
Side note, you could have done this addition in a separate pull-request. ;)

Reply:
Should have, yes